nltk/tokenize/stanford_segmenter.py

```diff
@@ -43,11 +46,17 @@ class StanfordSegmenter(TokenizerI):
     _JAR = 'stanford-segmenter.jar'
     _SLF4J = 'slf4j-api.jar'
 
-    def __init__(self, path_to_jar=None, path_to_slf4j=None,
-            path_to_sihan_corpora_dict=None,
-            path_to_model=None, path_to_dict=None,
-            encoding='UTF-8', options=None,
-            verbose=False, java_options='-mx2g'):
+    def __init__(self,
+                 path_to_jar=None, path_to_slf4j=None,
+                 java_class='edu.stanford.nlp.ie.crf.CRFClassifier',
+                 path_to_model=None,
+                 path_to_sihan_corpora_dict=None,
+                 sihan_post_processing='true',
+                 keep_whitespaces='false',
+                 path_to_dict=None,
+                 encoding='UTF-8', options=None,
+                 verbose=False, java_options='-mx2g'):
+
```

Comment:
Could you please add docstring saying what `path_to_sihan_corpora_dict`, `sihan_post_processing` and `keep_whitespaces` are.